### DISCUSSION TITLE
Problema 2: Duda Ejemplo 2

### DISCUSSION THREAD

**Discussion by Mateodlcc** [IIC2133-PUC/2024-2#167] [Tarea 3] [answered]
Hola! No me queda claro el segundo ejemplo de la parte 2. En este se indica que no se pueden conectar todos los maestros presentes (adjunto imagen).

![image](https://github.com/user-attachments/assets/a9a82eb3-01fa-48eb-82b5-7b35bbc023ac)

Mi duda está en el hecho de que si es que repetimos algunas conexiones, la información anterior es falsa, por ejemplo, si se realizan los entrenamientos de la siguiente forma:
1.- [1, 2]
2.- [2, 3]
3.- [3, 4] Aquí 3 y 4 quedan completos (saben todas las técnicas)
4.- [2, 3] De lo anterior, ahora 2 está completo
6.- [1, 2] Por último, 1 ahora sabe todas las técnicas

Lo anterior completa lo solicitado pero repitiendo ciertos entrenamientos.

Por lo anterior pregunto, deberíamos agregar esto como restricción? "Un entrenamiento se puede realizar máximo una vez"

Gracias!

**Comment by Mateodlcc** [comment]
Quizás lo que no me quedó claro fue lo que representa L. Es este valor la cantidad máxima de entrenamientos posibles para cumplir lo solicitado? O simplemente la cantidad de entrenamientos disponibles?

**Comment by Segismundios** [comment, marked as answer]
Holaa @Mateodlcc , tienes razón, sí se pueden conectar todos los maestros. Se va a subir una versión actualizada del enunciado.
Saludos!
